Documentation/basics.md

```diff
@@ -0,0 +1,136 @@
+# Trying out rkt 
+
+This guide provides a short introduction to trying out rkt. 
+For a more in-depth guide of a full end-to-end workflow of building an application and running it using rkt, check out the [getting-started-guide][getting-started-guide.md]
+
+## Using rkt on Linux
+
+`rkt` consists of a single self-contained CLI, and is currently supported on amd64 Linux.
+A modern kernel is required but there should be no other system dependencies.
+We recommend booting up a fresh virtual machine to test out rkt.
+
+To download the `rkt` binary, simply grab the latest release directly from GitHub:
+
+```
+wget https://github.com/coreos/rkt/releases/download/v0.13.0/rkt-v0.13.0.tar.gz
+tar xzvf rkt-v0.13.0.tar.gz
+cd rkt-v0.13.0
+./rkt help
+```
+
+## Trying out rkt using Vagrant
+
+For Mac (and other Vagrant) users we have set up a `Vagrantfile`: clone this repository and make sure you have [Vagrant](https://www.vagrantup.com/) 1.5.x or greater installed.
+`vagrant up` starts up a Linux box and installs via some scripts `rkt` and `actool`.
+With a subsequent `vagrant ssh` you are ready to go:
+
+```
+git clone https://github.com/coreos/rkt
+cd rkt
+vagrant up
+vagrant ssh
+```
+
+Keep in mind while running through the examples that right now `rkt` needs to be run as root for most operations.
+
+## rkt basics
+
+### Building App Container Images (ACIs)
+
+rkt's native image format is ACI, defined in the [App Container spec](app-container.md).
+To build ACIs, a simple way to get started is by using [`acbuild`](https://github.com/appc/acbuild).
+Another good resource is the [appc build repository](https://github.com/appc/build-repository) which has resources for building ACIs from a number of popular projects and languages.
```

Comment:
Want to use this opportunity to kill the build-repository?

I don't think there's anything in there worth salvaging, but if people want to still have an example on how to build a container using the host's apt-get I can add it into acbuild's examples.